localisation/00_zuwc_united_localisation.csv

```diff
@@ -220,11 +220,11 @@ EVTDESC_WCBDP_60_alliance;The Alliance Expedition has crossed through the veil o
 EVTOPTA_WCBDP_60_alliance;It is foolish to rush forward at breakneck speed...;;Es w�re t�richt, kopflos auf den Gegner loszust�rmen...;;;;;;;;;;;x
 EVTOPTA_WCBDP_60_tooltip;You try to gain a foothold on the peninsula.\n;;Ihr versucht, einen Au�enposten auf der Halbinsel zu etablieren.\n;;;;;;;;;;;x
 EVTOPTB_WCBDP_60_alliance;What nonsense?! Take this fortress!;;Was f�r ein Unsinn! Nehmt diese Festung!;;;;;;;;;;;x
-EVTOPTB_WCBDP_60_tooltip;Your troops take the Citadel\n;;Eure Truppen nehmen die Zitadelle ein.\n;;;;;;;;;;;x
+EVTOPTB_WCBDP_60_tooltip;Your troops take the Citadel.\n;;Eure Truppen nehmen die Zitadelle ein.\n;;;;;;;;;;;x
```

Comment:
One thing, it makes sense to remove `\n`. Does nothing.

Reply:
Done. Didn't want to change it since I didn't know why it was there in the first place. Probably it was longer and got shortened to this at some place.